src/main/java/com/rocketden/main/dto/game/GameDto.java

```diff
@@ -0,0 +1,16 @@
+package com.rocketden.main.dto.game;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Map;
+
+import com.rocketden.main.dto.room.RoomDto;
+
+@Getter
+@Setter
+public class GameDto {
```

Comment:
Reminder note: I think we touched on it during the call that the structure of the game DTO's might change or be broken up for different subscribe endpoints, but this looks good for now

Reply:
Yeah absolutely, I think a lot of this will morph going forward, but hopefully this serves as a solid basis from which to develop.